src/modules/organizations/components/categories-editor.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import DragReorderable from 'drag-reorderable';
+
+export default class CategoriesEditor extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleAddCategory = this.handleAddCategory.bind(this);
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleCategoryReorder = this.handleCategoryReorder.bind(this);
+    this.handleRemoveCategory = this.handleRemoveCategory.bind(this);
+    this.renderRow = this.renderRow.bind(this);
+    this.renderTable = this.renderTable.bind(this);
+  }
+
+  handleAddCategory() {
+    const newCategory = { key: Math.random(), category: 'Example' };
+    const categories = this.props.categories;
+    categories.push(newCategory);
+    this.props.onChange(categories);
+  }
+
+  handleInputChange(idx, event) {
+    const categories = this.props.categories;
+    categories[idx].category = event.target.value;
```

Comment:
Variable name choice: maybe `categories[idx].name` or `.title` instead of `categories[idx].category` would make it a bit more clear what this represents?